src/components/containers.js

```diff
@@ -4,30 +4,48 @@
 
 import classnames from 'classnames';
 
+import { refElement } from '../util/typing';
+
 /**
  * @typedef {import('preact').ComponentChildren} Children
  *
  * @typedef ContainerProps
  * @prop {Children} children
  * @prop {string} [classes] - Additional CSS classes to apply
+ * @prop {import('preact').Ref<HTMLElement>} [containerRef] - Access to the
```

Comment:
What are the use cases you have in mind for `containerRef`?

Reply:
E.g. https://github.com/hypothesis/frontend-shared/pull/174/files#diff-39adc517b1457f125e1e729d1039b7ce6a3a4c39317a18af693ff418d5cf6073R153